### DISCUSSION TITLE
Exception at runtime with forge

### DISCUSSION THREAD

**Discussion by Lythom** [shedaniel/RoughlyEnoughItems#901] [Q & A] [answered]
Hello !
I'm trying to upgrade my mod ([capsule](https://github.com/Lythom/capsule/)) to 1.19 and I'm trying to replace jei by rei to do so.
After migrating all my code , if I disable jeiPlugin and doesn't load any 3rd party jarit runs fine.
But whenever I add `implementation fg.deobf("me.shedaniel:RoughlyEnoughItems-forge:9.0.472")` + the matching maven entry I get an error during lunch : https://mclo.gs/oTJaS9e
Here is the build.grade file i use: https://github.com/Lythom/capsule/blob/666717f9fb27b58eae954db29e176495971ffb3b/build.gradle

I tried with forge 41.0.14 and 41.0.5. 
Is RoughlyEnoughItems designed to work around specific versions of forge ?
Any idea of what is wrong and how I could fix it ?

**Comment by shedaniel** [comment, marked as answer]
You may follow https://docs.architectury.dev/docs/forge_loom/fg_broken_refmap.html

You might also want to add architectury api and cloth config into the dev environment. You can do this via https://linkie.shedaniel.me/dependencies

---
If this answer helped you, please mark this as the answer.

**Reply by Lythom** [reply]
Okay, I actually have no idea of why it works but adding `mixin.env.refMapRemappingFile` got it working. Thanks for the help !
